src/main/java/be/seeseemelk/mockbukkit/MockPlayerList.java

```diff
@@ -95,6 +106,13 @@ public boolean isSomeoneOnline()
 		return !onlinePlayers.isEmpty();
 	}
 
+	public boolean isPlayerAlreadyOnServer(@NotNull PlayerMock player)
```

Comment:
I think this is a more reasonable name.
```suggestion
	public boolean isOnline(@NotNull PlayerMock player)
```

Reply:
xd